File: Kattis/Tarifa.py
import sys

godlist = []

for i in sys.stdin:
    godlist.append(int(i))

mblimit = godlist[0]
months = godlist[1]
output = 0

for j in range(2,len(godlist)):
    output += (mblimit - godlist[j])

output += mblimit
print(output)

File: Kattis/tarifa.cpp
#include <iostream>
using namespace std;

int main(){

  long mblimit;
  long months;
  long mbused;
  long output = 0;

  cin >> mblimit;
  cin >> months;

  while (cin >> mbused){
    output += (mblimit - mbused);
  }

  output += mblimit;

  cout << output;

}